convert/convert.go

```diff
@@ -151,40 +153,57 @@ func (c *Converter) Convert(wa io.WriterAt, ra io.ReaderAt, size int64) error {
 				}
 
 				for start < end {
-					// The last read may be shorter.
-					n := len(buf)
-					if end-start < int64(len(buf)) {
-						n = int(end - start)
+					// Get next extent in this segment.
```

Comment:
Needs definition of "segment", "area", "extent", "cluster"

Reply:
I improved the comments, I hope it is clear enough now.